frontend/src/components/Tree/Tree.jsx

```diff
@@ -48,6 +48,7 @@ function renderTree(dimensions, jsonData, svgRef, onNodeClick) {
   const { width, height } = dimensions;
 
   const radius = Math.min(width, height);
+  const translateStr = "translate(" + width / 2 + "," + height / 2 + ")";
```

Comment:
For this line, how did you know to do width, height / 2 instead of anything else? Is this just to center it?

Reply:
Yup just to center it. "width" is supposed to represent the full page width so dividing by 2 would put the center node of the tree at the center of the page.